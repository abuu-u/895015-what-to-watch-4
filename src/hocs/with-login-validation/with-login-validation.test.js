import React from "react";
import renderer from "react-test-renderer";
import withLoginValidation from "./with-login-validation.js";

const MockComponent = () => <div/>;

const MockComponentWrapped = withLoginValidation(MockComponent);

it(`withFilmPlayer is rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped
      onLogin={() => {}}
    />
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
