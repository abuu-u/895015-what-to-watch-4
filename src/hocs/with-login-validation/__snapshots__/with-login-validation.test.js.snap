// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withFilmPlayer is rendered correctly 1`] = `<div />`;
